skdecide/hub/domain/gym/gym.py

```diff
@@ -1192,7 +1204,10 @@ def close(self):
 
         Environments will automatically close() themselves when garbage collected or when the program exits.
         """
-        return self._domain.close()
+        # check that the method "close" exists before calling it. (For instance Maze domain do not has one.)
```

Comment:
Just a minor typo suggestion:

```suggestion
        # check that the method "close" exists before calling it (for instance the Maze domain does not have one).
```